listening-comp/backend/example_usage.py

```
@@ -0,0 +1,160 @@
+"""
+Example script demonstrating how to use the database operations and models.
+This file shows common usage patterns for the CRUD operations.
+"""
+import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)